client/oidc.go

```diff
@@ -89,6 +99,52 @@ func VerifyPKToken(ctx context.Context, pkt *pktoken.PKToken, provider OpenIdPro
 	return nil
 }
 
+func DiscoverPublicKey(ctx context.Context, headersEnc []byte, issuer string) (crypto.PublicKey, error) {
```

Comment:
I've tried to use a postfix of `B64` to signal that a variable is Base64 encoded and I'd like to maintain that as a convention: `headersEnc` to `headersB64`.